poms/bom-test/pom.xml

```diff
@@ -404,6 +404,11 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            <dependency>
+                <groupId>org.xmlunit</groupId>
+                <artifactId>xmlunit-core</artifactId>
+                <version>${xmlunit.version}</version>
+            </dependency>
```

Comment:
Please add this to the list of test dependencies in the Dependabot config.

Reply:
done :+1: